### PR TITLE
Handle noindex directive

### PR DESCRIPTION
**Pull request by LeMoussel**
Resolve Issue #26

### CODE REVIEW

File: RobotsTxtParser.php [hunk 1]
```diff
@@ -255,14 +264,15 @@ public function prepareRules()
 	 */
 	private function shouldSwitchToZeroPoint()
 	{
-		return in_array(strtolower($this->current_word), array(
+		return in_array(strtolower($this->current_word, array(
```
Comment (bopoda) [inline RobotsTxtParser.php]: Please, close ")" after $this->current_word. There an error according to phpunit tests:
```
Parse error: syntax error, unexpected ';' in /home/travis/build/bopoda/robots-txt-parser/RobotsTxtParser.php on line 276
```

**Comment by bopoda** [comment]
Thanks for PR!
looks good, but could you please to fix a few comments above? It should be merged when travis build will be ok.

File: RobotsTxtParser.php (hunk 1 above)
Comment (LeMoussel) [inline RobotsTxtParser.php]: OK, 
I'm newest with Git. How can do this with this PR? Should I do a new PR?

**Comment by bopoda** [comment]
really, thanks. Then it is a bug which should be fixed.

**Comment by bopoda** [comment]
Tests passed. 
Thanks!

**Pull request merged**